applications/minotari_ledger_wallet/wallet/src/main.rs

```diff
@@ -87,21 +89,24 @@ unsafe impl critical_section::Impl for MyCriticalSection {
 // Application status words.
 #[repr(u16)]
 pub enum AppSW {
```

Comment:
declared twice?

Reply:
I created a seperate crate for common types shared between the Ledger application and the rest of the code base.